data/noiseprofiles.json

```diff
@@ -2647,6 +2647,37 @@
           ]
         },
         {
+          "comment": "dmc-gm5 contributed by axel burri",
```

Comment:
The file can be downloaded from here:
http://dev.tty0.ch/dt-noiseprofile-20150425.tar.gz